src/test/resources/org/rust/ide/intentions/fixtures/unwrap_lambda_expr/unwrap_braces.rs

```diff
@@ -0,0 +1,3 @@
+fn main () {
+    |x| { x *<caret> x }
```

Comment:
Oups, looks like I've accidentally edited @mkaput comment instead of creating a reply :sweat_smile: https://github.com/intellij-rust/intellij-rust/pull/576#discussion_r73699058

Sorry for that.